src/library_pthread.js

```diff
@@ -571,6 +572,13 @@ var LibraryPThread = {
     else postMessage({ 'cmd': 'cleanupThread', 'thread': thread });
   },
 
+  _emscripten_thread_set_strongref: function(thread) {
```

Comment:
Perhaps a comment here about how the sole current use case for this function is for keeping the "main" thread alive in PROXY_TO_PTHREAD mode?

Reply:
Addressed with 8f2bc84fc8839dead3cf46b896a3673bd7bd1578.